src/dwn.ts

```diff
@@ -113,11 +111,11 @@ export class Dwn {
     console.groupEnd();
 
     console.group('messageStore');
-    await this.messageStore['dump']?.();
+    await this.messageStore['dump']?.(); // FIXME: messageStore.get requires cid and other bits
```

Comment:
See my other comment about `dump()`. This is fine as is. We should just ts-ignore